### PR TITLE
(#862) Add information about audit logs page

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

This replaces the Work in Progress page that was left here. 

## Motivation and Context

Having a work in progress section isn't very helpful

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [x] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Relates to #862

### CODE REVIEW

File: input/en-us/central-management/usage/website/administration/maintenance.md [hunk 1]
```diff
@@ -5,6 +5,34 @@ Title: Maintenance
...
+It is possible to see all available information for an operation by clicking the :mag: button to the left-hand side of the table.
+
+If required, the available results can be exported to an Excel document using the `Export | Export to Excel` button.
+This will generate a file named something similar to the following `Chocolatey_AuditLogs_20231122_095144.xlsx`.
+
+The length of time that Operation logs are stored within the Chocolatey Central Management Website can be configured in the [Retention Policies](xref:ccm-administration-settings-retention#audit-retention) Settings tab.
+
+## Changes logs
+
+This section of the Chocolatey Central Management Website does not actually provide any functionality, and will be removed in a future version.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/usage/website/administration/maintenance.md:38]: Should this maybe be a warning callout, instead of just flat text?

Comment (gep13) [reply input/en-us/central-management/usage/website/administration/maintenance.md:38]: Yes, I think that is a good idea!  I will get this fixed up!

**Review by AdmiringWorm — APPROVED** [review]
LGTM, the only thing left is the to add the image.

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Pull request merged**